accounts/abi/reflect_test.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
2019 :-p

Reply:
I really have a hard time moving into the new year...